build/ci/phase-template.yml

```diff
@@ -36,5 +36,25 @@ phases:
         testRunTitle: Machinelearning_Tests_$(_phaseName)_$(_configuration)_$(Build.BuildNumber)
         configuration: $(_configuration)
         mergeTestResults: true
+    - task: CopyFiles@2
+      displayName: Stage build logs
+      inputs:
+        sourceFolder: $(Build.SourcesDirectory)
+        contents: 
+          ?(msbuild.*|binclash.log|init-tools.log)
+        targetFolder: $(Build.ArtifactStagingDirectory)
+    - task: CopyFiles@2
+      displayName: Stage more build logs
+      inputs:
+        sourceFolder: $(Build.SourcesDirectory)
+        contents: 
+          bin/**/Microsoft.ML.Predictor.Tests/netcoreapp2.1/TestOutput/**/*
```

Comment:
Why only Predictor.Tests?

Do we know what files are under TestOutput? Maybe we could avoid duplicating this task twice and do it all at once by expanding the contents logic.

Reply:
I was specifying Predictor.Tests because right now that is the only one that has test output logs.  I like the idea of just letting it generically be "TestOutput" to cover future cases where we add more logs.  I'll change that.